File: exams/frmLoading.vb
Public Class frmLoading
    Dim i As Integer = 0
    Private Sub frmLoading_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        StartTimer.Enabled = True
    End Sub

    Private Sub StartTimer_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles StartTimer.Tick
        progressBar.Increment(5)
        i += 2
        If i >= 120 Then
            StartTimer.Enabled = False
            Me.Close()
        End If
    End Sub
End Class

File: exams/frmLoading.cs
using System;

namespace exams
{
    public partial class frmLoading
    {
        public frmLoading()
        {
            InitializeComponent();
        }

        private int i = 0;

        private void frmLoading_Load(object sender, EventArgs e)
        {
            StartTimer.Enabled = true;
        }

        private void StartTimer_Tick(object sender, EventArgs e)
        {
            progressBar.Increment(5);
            i += 2;
            if (i >= 120)
            {
                StartTimer.Enabled = false;
                Close();
            }
        }
    }
}